src/components/Header/index.js

```diff
@@ -16,41 +16,51 @@ import List from 'material-ui-icons/List'
 import Search from 'material-ui-icons/Search'
 import Add from 'material-ui-icons/Add'
 import HelpOutline from 'material-ui-icons/HelpOutline'
+import Person from 'material-ui-icons/Person'
 
 import {Link} from 'react-router-dom'
 
-import cityOfHelsinkiLogo from 'src/assets/images/helsinki-coat-of-arms-white.png'
+import cityOfHelsinkiLogo from 'src/assets/images/helsinki-logo.1.svg'
 
 class HeaderBar extends React.Component {
 
     render() {
         let buttonStyle = {color: '#ffffff'}
```

Comment:
use sass variable, same with buttonStyle2
buttonStyle2 is also a bad var name. No inline style is needed here, should remove